fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1355,6 +1356,49 @@ public Map<Table, SlotRef> getTableToPartitionSlotMap() {
         return tableToJoinExprMap;
     }
 
+    // In Loose mode, do not need to check mv partition's data is consistent with base table's partition's data.
+    // Only need to check the mv partition existence.
+    public boolean getPartitionNamesToRefreshForMvInLooseMode(Set<String> toRefreshPartitions) {
+        Expr partitionExpr = getFirstPartitionRefTableExpr();
+        Map<Table, Column> partitionTableAndColumn = getRelatedPartitionTableAndColumn();
+        if (partitionExpr == null || partitionTableAndColumn.isEmpty()) {
+            return this.getPartitions().size() > 0;
+        }
+        Map<String, Range<PartitionKey>> mvRangePartitionMap = getRangePartitionMap();
```

Comment:
is this method safe for unpartitioned mv?  add a unpartitioned mv case?

Reply:
I think unpartitioned mv colud be deal with line:1364